www/components/Button.js

```diff
@@ -63,9 +69,12 @@ const Button = ({ icon, text, size, onClick, varient, stretch }) => {
     break;
   case false:
     break;
+  case true:
+    className += " " + styles.stretchTrue;
+    break;
   }
-  //icons
 
+  //icons
```

Comment:
It would be cool here to check if what is being passed is an icon component so that a user can put in what they want.

Reply:
good idea, will implement